foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -1,1182 +1,1805 @@
+{
+  protected override void OnCreate(Bundle savedInstanceState)
+  {
+    base.OnCreate(savedInstanceState);
+
+    global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
+    LoadApplication(new App());
+
+    // set the wrapper type
+    ACPCore.SetWrapperType(WrapperType.Xamarin);
+
+    // register SDK extensions
+    ACPCore.Application = this.Application;
+    ACPIdentity.RegisterExtension();
+    ACPLifecycle.RegisterExtension();
+    ACPSignal.RegisterExtension();
+
+    // start core
+    ACPCore.Start(null);
+}
+```
+{% endtab %}
+{% endtabs %}
+
+## registerURLHandler 
+
+Mobile SDK allows you to add a callback function that is triggered before the [`open url`](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine#consequence-types) action occurs. If the callback function returns **Yes**, the SDK does not complete the `open url` action. If the callback function returns **No**, the SDK completes the `open url` action.
+
+{% tabs %}
+{% tab title="iOS \(ACP 2.x\)" %}
+**Objective C**
+
+### registerURLHandler
+
+**Syntax**
+
+```objectivec
++ (void) registerURLHandler: (nonnull BOOL (^) (NSString* __nullable url)) callback;
+```
+
+**Example**
+
+```objectivec
+[ACPCore registerURLHandler:^BOOL(NSString * _Nullable url) {
+    ...
+}];
+```
+{% endtab %}
+{% endtabs %}
+
+## resetIdentities
+
+The `resetIdentities` method requests that each extension resets the identities it owns. Each extension responds to this request uniquely.
+
+{% hint style="warning" %}
+This API is only handled by the [Edge](mobile-core-api-reference.md) and [Edge Identity](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/de62274f59c14b23acae8992352ec63fbb16411b/foundation-extensions/adobe-edge-identity/README.md) extensions. Calling `resetIdentities` does **not** clear identities in non-edge extensions, such as [Identity](identity/).
+{% endhint %}
+
+{% tabs %}
+{% tab title="Android" %}
+**Java**
+
+### resetIdentities
+
+Note, this method is only available in Mobile Core v.1.8.0 and above.
+
+**Syntax**
+
+```java
+void resetIdentities();
+```
+
+**Example**
+
+```java
+MobileCore.resetIdentities();
+```
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+**Swift**
+
+### resetIdentities
+
+**Syntax**
+
+```swift
+static func resetIdentities()
+```
+
+**Example**
+
+```swift
+MobileCore.resetIdentities()
+```
+
+**Objective-C**
+
+### resetIdentities
+
+**Syntax**
+
+```objectivec
+@objc(resetIdentities)
+static func resetIdentities()
+```
+
+**Example**
+
+```objectivec
+[AEPMobileCore resetIdentities];
+```
+
+{% endtab %}
+{% endtabs %}
+
+## setAdvertisingIdentifier
+
+The advertising ID is preserved between app upgrades, is saved and restored during the standard application backup process, available via [Signals](signals/), and is removed at uninstall.
+
+For more information about identity in Mobile Core, please read the documentation on the [identity APIs](identity/identity-api-reference.md).
```

Comment:
can we update link to (identity/identity-api-reference.md#setadvertisingidentifier) ?